tutorials-and-examples/workflow-orchestration/dws-examples/README.md

```diff
@@ -1,11 +1,20 @@
 # Dynamic Workload Scheduler examples
 
-The repository contains examples on how to use DWS in GKE. More information about DWS is 
+The repository contains examples on how to use DWS in GKE. More information about DWS is
 available [here](https://cloud.google.com/kubernetes-engine/docs/how-to/provisioningrequest).
 
-Files included:
+## Prerequisites:
+
+### [Kueue](https://kueue.sigs.k8s.io/)
+To install a released version of Kueue in your cluster, run the following command:
+
+```sh
+VERSION=v0.7.0
+kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
```

Comment:
Also link to https://kueue.sigs.k8s.io/docs/installation/ for more configuration options

Reply:
Done